tests/test_tests.py

```diff
@@ -40,3 +40,23 @@ def test_raospacing():
     assert_allclose(0.001, p, atol=0.0001, rtol=0.001)
     assert_allclose(233.7789, U, atol=0.001, rtol=0.001)
     assert_allclose(192.17, Uc, atol=0.001, rtol=0.001)
+    
+def test_vtest():
+    data = np.array([
+        -1.78277804,  0.20180845, -1.51291097,  0.57329272,  0.72195959,
+        1.59947271,  1.4666837, -0.36532379,  1.4455209,  0.77365236,
+        5.8678466,  2.58327349,  0.25429634,  1.74902778,  4.13215085,
+        0.20612467,  1.38484181,  1.72546928,  3.33570062,  3.96191276])
+    p, V = pycircstat.tests.vtest(data,0)
+    assert_allclose(0.1074, p, atol=0.001, rtol=0.001)
+    assert_allclose(3.9230, V, atol=0.001, rtol=0.001)
+    
+def test_symtest():
+    data = np.array([
+        -1.78277804,  0.20180845, -1.51291097,  0.57329272,  0.72195959,
+        1.59947271,  1.4666837, -0.36532379,  1.4455209,  0.77365236,
+        5.8678466,  2.58327349,  0.25429634,  1.74902778,  4.13215085,
+        0.20612467,  1.38484181,  1.72546928,  3.33570062,  3.96191276])
+    p = pycircstat.tests.symtest(data)
```

Comment:
`symtest` returns two values: `pval, T`. If you change this line to `p, T = pycircstat.tests.symtest(data)`, the test should pass (but you are not checking the value of T, maybe we should add a second assertion for that?)

Best,
  Matthias